docs/tutorial/cpp/src/getting_started.md

```diff
@@ -1,88 +1,82 @@
 <!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
 # Getting Started
 
-In this tutorial, we use C++ as the host programming language. We also support other programming languages like
+This tutorial uses C++ as the host programming language. Slint also supports other programming languages like
 [Rust](https://slint.dev/docs/rust/slint/) or [JavaScript](https://slint.dev/docs/node/).
 
-You will need a development environment that can compile C++20, [CMake 3.21](https://cmake.org/download/),
+Slint has an application template you can use to create a project with dependencies already set up that follows recommended best practices.
+
+Before using the template, you need a C++ compiler that supports C++ 20 and to install [CMake](https://cmake.org/download/) 3.21 or newer.
+
+Clone or download template repository:
+
+```sh
+git clone https://github.com/slint-ui/slint-cpp-template memory
+cd memory
+```
+
+<!-- You will need a development environment that can compile C++20, [CMake 3.21](https://cmake.org/download/),
```

Comment:
Since the old bit is commented out, I guess it would be fine to also remove it entirely :)

Reply:
@tronical Yeah I kind of left it there in case it turned out later that I needed it after testing, but seemingly not.